Jenkinsfile

```
@@ -0,0 +1,71 @@
+#!groovy
+
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+  }
+
+  stages {
+    stage('Build Docker image') {
+      agent any
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/zoomapper'
+          def dockerImageName = "${dockerRepoName}:${GIT_COMMIT}"
+          def newImage = docker.build(dockerImageName)
+
+          if (BRANCH_NAME == 'master') {
+            stage('Update latest tag') {
+              newImage.push()
+              newImage.push('latest')
+            }
+          }
+        }
+      }
+    }
+
```

Add in a build check on the k8s configs, this will just ensure these are valid syntax etc and won't fail on deploy / merge, https://github.com/zooniverse/panoptes/blob/f6e97acf4d9c36f1ec082aeb65f1d601a419c3c1/Jenkinsfile#L49-L57
```suggestion

stage('Dry run deployments') {
      agent any
      steps {
        sh "sed 's/__IMAGE_TAG__/${GIT_COMMIT}/g' kubernetes/deployment-staging.tmpl | kubectl --context azure apply --dry-run=client --record -f -"
        sh "sed 's/__IMAGE_TAG__/${GIT_COMMIT}/g' kubernetes/deployment-production.tmpl | kubectl --context azure apply --dry-run=client --record -f -"
      }
    }

```